### PR TITLE
Add Stingray Music Connector

### PR DESCRIPTION
**Pull request by jbwharris**
**Describe the changes you made**
Create connector for Stingray Music

**Additional context**
https://www.stingray.com/
This one works well if you're using an ad blocker like uBlock Origin, but I had noted issues with being a track behind when it did ad insertion that I wasn't able to figure out how to reconcile that.

### CODE REVIEW

**Review by alexesprit — CHANGES_REQUESTED** [review]
This website contains additional information you could add to the connector:
1. Album name
2. Cover art
3. Play/pause button

File: src/core/connectors.js [hunk 1]
```diff
@@ -1740,6 +1740,13 @@ const connectors = [{
 	],
 	js: 'connectors/turntable.fm.js',
 	id: 'turntable.fm',
+}, {
+	label: 'Stingray Music',
+	matches: [
+		'*://webplayer.stingray.com/*',
```
Comment (alexesprit) [inline src/core/connectors.js:1746]: I would replace this pattern with `*.stingray.com/*` to let the extension mark the main website (stingray.com) as supported.

**Comment by jbwharris** [comment]
@alexesprit I have made the suggested changes.

File: src/connectors/stingray.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+Connector.playerSelector = 'player';
+Connector.artistSelector = 'cover-with-info.current.cover artist-names div.value';
+Connector.trackSelector = 'cover-with-info.current.cover div.title-content > span.title';
+Connector.albumSelector = 'cover-with-info.current.cover div.album';
+Connector.getTrackArt = () => Util.extractUrlFromCssProperty(document.querySelector('cover-with-info.current.cover div.background').style['background-image']);
```
Comment (alexesprit) [inline src/connectors/stingray.js:7]: Default implementation has already this logic. Using `Connector.trackArtSelector` property is enough here.

**Review by alexesprit — APPROVED** [review]

**Pull request merged**